subsys/bluetooth/services/fast_pair/CMakeLists.txt

```diff
@@ -39,59 +39,10 @@ if(CONFIG_BT_FAST_PAIR_FMDN)
   target_link_libraries(fast_pair PRIVATE fmdn)
 endif()
 
-if(SYSBUILD)
   zephyr_get(FP_DATA_PRESENT MERGE SYSBUILD)
 
-  if(NOT FP_DATA_PRESENT AND TARGET zephyr_interface)
-    zephyr_compile_definitions(
-      FP_DATA_NOT_PRESENT
-      )
-  endif()
-endif()
-
-if(NOT SYSBUILD)
-  set(FP_PARTITION_NAME bt_fast_pair)
-
-  set(
-    FP_PROVISIONING_DATA_HEX
-    ${CMAKE_BINARY_DIR}/modules/nrf/subsys/bluetooth/services/fast_pair/fp_provisioning_data.hex
-    )
-
-  set(FP_PROVISIONING_DATA_ADDRESS $<TARGET_PROPERTY:partition_manager,PM_BT_FAST_PAIR_ADDRESS>)
-
-  if(NOT DEFINED FP_MODEL_ID OR NOT DEFINED FP_ANTI_SPOOFING_KEY)
-    message(FATAL_ERROR "FP_MODEL_ID and FP_ANTI_SPOOFING_KEY build variables must be specified.")
-  endif()
-
-  add_custom_command(
-    OUTPUT
-    ${FP_PROVISIONING_DATA_HEX}
-    DEPENDS
-    "${CMAKE_BINARY_DIR}/pm.config"
-    COMMAND
-    ${PYTHON_EXECUTABLE} ${ZEPHYR_NRF_MODULE_DIR}/scripts/nrf_provision/fast_pair/fp_provision_cli.py
-                         -o ${FP_PROVISIONING_DATA_HEX} -a ${FP_PROVISIONING_DATA_ADDRESS}
-                         -m ${FP_MODEL_ID} -k ${FP_ANTI_SPOOFING_KEY}
-    COMMENT
-    "Generating Fast Pair provisioning data hex file"
-    USES_TERMINAL
-    )
-
-  add_custom_target(
-    ${FP_PARTITION_NAME}_target
-    DEPENDS
-    "${FP_PROVISIONING_DATA_HEX}"
-    )
-
-  set_property(
-    GLOBAL PROPERTY
-    ${FP_PARTITION_NAME}_PM_HEX_FILE
-    "${FP_PROVISIONING_DATA_HEX}"
-    )
-
-  set_property(
-    GLOBAL PROPERTY
-    ${FP_PARTITION_NAME}_PM_TARGET
-    ${FP_PARTITION_NAME}_target
+if(NOT FP_DATA_PRESENT AND TARGET zephyr_interface)
```

Comment:
Generating Fast Pair provisioning data without sysbuild seems not working (seems related to using PM for single image builds without sysbuild: `CONFIG_PM_SINGLE_IMAGE`). Try `ncs/nrf/samples/bluetooth/fast_pair/input_device` built as `west build -b nrf52840dk/nrf52840 -p --no-sysbuild`.

Should we allow users to enable `CONFIG_PM_SINGLE_IMAGE` without sysbuild? Maybe we need a dependency here? Or maybe we can remove the option as IIRC PM is enabled for sysbuild by default?

Reply:
Have removed the Kconfig